pysat/utils/coords.py

```diff
@@ -101,6 +110,17 @@ def calc_solar_local_time(inst, lon_name=None, slt_name='slt'):
     if lon_name not in inst.data.keys():
         raise ValueError('unknown longitude variable name')
 
+    if ref_date is not None and apply_modulus:
+        estr = 'Keyword `ref_date` only supported if `apply_modulus`=False.'
+        raise ValueError(estr)
```

Comment:
I think this should at worst be a warning, though I would rank it at logging.info.  Supplying unused info isn't going to hurt anyone.